requirements2.txt

```diff
@@ -0,0 +1,18 @@
+six==0.1.10
+mock==1.0.1
+biopython==1.64
+bz2file==0.98
+cython==0.21.2
+discover==0.4.0
+ipython==3.1.0
+matplotlib==1.4.3
+numpy==1.8.1
+pandas==0.14.1
+pep8==1.5.7
+pyflakes==0.8.1
+pyzmq==14.3.1
+reportlab==3.1.44
+simplejson==3.5.3
+tornado==4.0.1
+--allow-external mysql-connector-python
+mysql-connector-python==2.0.3
```

Comment:
Would it make sense to say somewhere when which requirements file should be used?